swarm-derive/tests/test.rs

```diff
@@ -133,6 +133,23 @@ fn custom_event() {
     }
 }
 
+#[test]
+fn custom_prefix() {
+    use libp2p as mylibp2p;
+    #[allow(dead_code)]
+    #[derive(NetworkBehaviour)]
+    #[behaviour(libp2p_prefix = "mylibp2p")]
```

Comment:
Serde calls this parameter `crate`: https://serde.rs/container-attrs.html#crate

I'd be in favor of following that here too.

Reply:
OK, done